coretex/bioinformatics/__init__.py

```diff
@@ -15,4 +15,35 @@
 #     You should have received a copy of the GNU Affero General Public License
 #     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from .utils import CommandException
+from typing import Optional
+
+from .utils import CommandException, command
+
+
+def cutadaptTrim(
+    forewardFile: str,
```

Comment:
why are we using `str` instead of `Path` for paths?

Reply:
Requiring strings is the convention that all other command wrapper functions in the bioinformatics subdirectory use, so this one should not be an exception

Reply:
For now I will resolve this but this needs to be changed